examples/01-Getting-started.ipynb

```diff
@@ -0,0 +1,1012 @@
+{
```

Comment:
<p>I think the the outline looks great, but I would change it:</p><p> </p><p>Headings:</p><ul><li>Downloading the dataset</li><li>Processing the dataset with NVTabular</li><li>Training a DLRM model</li></ul><p> </p><p>The current headings (e.g. defining the workflow) can be subheadings below Processing the dataset with NVTabular</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/NVTabular/pull/1633/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>made the change</p>